styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -460,28 +461,6 @@ private void setupMetrics(
     stats.registerWorkflowCount("enabled", configuredEnabledWorkflowsCount);
   }
 
-  private TriggerListener trigger(
-      Storage storage,
-      StateFactory stateFactory,
-      StateManager stateManager) {
-    final TriggerListener stateInitializingTrigger =
-        new StateInitializingTrigger(stateFactory, stateManager, storage);
-
-    return (workflow, trigger, instant) -> {
-      try {
-        final boolean enabled = storage.workflowState(workflow.id()).enabled().orElse(false);
```

Comment:
Looks like this per-workflow enable check was dropped? This would lead to disabled workflows triggering from the TriggerManager.